pom.xml

```
@@ -235,7 +235,8 @@
 				<configuration>
 				    <target>
 						<file>../target-platform/GEF_classic.target</file>
-					  </target>
+					</target>
+				    <executionEnvironment>JavaSE-21</executionEnvironment>

```

Just some minor formatting:

```suggestion
				    <executionEnvironment>JavaSE-21</executionEnvironment>
					<executionEnvironment>JavaSE-21</executionEnvironment>
```

---

Thx for noting fixed everything with tabs.